types/koa-bunyan-logger/koa-bunyan-logger-tests.ts

```diff
@@ -0,0 +1,12 @@
+import * as Koa from 'koa';
+import koaBunyanLogger from 'koa-bunyan-logger';
```

Comment:
You should use `import name = require('package');` instead.

If you still want to use the syntax `import name from 'package';` you need to use `esModuleInterop` on the compiler settings from your consuming code.